pac4j-saml/src/main/java/org/pac4j/saml/crypto/ExplicitSignatureTrustEngineProvider.java

```diff
@@ -17,11 +26,20 @@
  * @since 1.8.0
  */
 public class ExplicitSignatureTrustEngineProvider implements SAML2SignatureTrustEngineProvider {
-
+    
+    private static final Logger log = LoggerFactory.getLogger(ExplicitSignatureTrustEngineProvider.class);
+    
     private final MetadataResolver metadataResolver;
+    
+    private final boolean signatureValidationDisabled;
 
     public ExplicitSignatureTrustEngineProvider(final MetadataResolver metadataResolver) {
+        this(metadataResolver, false);
+    }
+    
+    public ExplicitSignatureTrustEngineProvider(final MetadataResolver metadataResolver, final boolean signatureValidationDisabled) {
```

Comment:
Please inject the engine from the outside, rather than a boolean parameter to build the engine internally.